docs/markdown/Getting Started/getting-started/installation.md

```diff
@@ -6,13 +6,15 @@ createdAt: "2020-02-21T17:44:53.022Z"
 updatedAt: "2022-07-12T00:00:00.000Z"
 ---
 
-To install the `pants` binary you can use:
+You can download an installer script that will install the Pants binary with this command:
 
 ```
-/bin/bash -c "$(curl -fsSL https://static.pantsbuild.org/setup/pantsup.sh)" 
+curl --proto '=https' --tlsv1.2 -fsSL https://static.pantsbuild.org/setup/pantsup.sh
```

Comment:
This will dump the script to the console. Are you expecting they know how to redirect to a file or did you intend -O but miss it?

I will say having the checked in script start with pants is unfortunate. Anyone with `.` on their PATH may swear at you.

Reply:
I assumed people would know how to redirect it. I can add `-O` for extra handholding.

Why will those folks swear at me? Because of tab-completion?

Reply:
Exactly.